be/src/exec/base_scanner.cpp

```diff
@@ -154,10 +154,10 @@ bool BaseScanner::fill_dest_tuple(const Slice& line, Tuple* dest_tuple, MemPool*
                 ctx->clear_error_msg();
                 return false;
             }
-            SlotDescriptor* slot_descriptor = _src_slot_descs_order_by_dest[dest_index];
-            if (_strict_mode && (slot_descriptor != nullptr)&& !_src_tuple->is_null(slot_descriptor->null_indicator_offset())) {
+            if (_strict_mode && (_src_slot_descs_order_by_dest[dest_index] != nullptr) &&
```

Comment:
better leave a comment for this change, I'm afraid other will change it back someday.

Reply:
OK